Update user guide for Traceflow

antoninbas opened the pull request:
* UI now supports a Service as the destination
 * In order to use Service as the destination, AntreaProxy should be
   enabled

jianjuns commented on docs/traceflow-guide.md:
```diff
@@ -27,24 +31,27 @@ You need to switch on traceflow from featureGates defined in antrea.yml for both
     featureGates:
     # Enable traceflow which provides packet tracing feature to diagnose network issue.
       Traceflow: true
+    # Enable antrea proxy which provides ServiceLB for in-cluster services in antrea agent.
```
I saw we use both "Antrea" and "antrea". I thought it should be "Antrea".

jianjuns replied:
services -> Services

antoninbas replied:
Will make the change. This is taken straight from the antrea-agent.conf file, so I'll update it everywhere.

antoninbas replied:
Done

antoninbas replied:
Done, thanks

mengdie-song reviewed (commented):
Thanks for the change. I think the yaml definition and octant screenshot for `Start New Trace`  needs to be updated with the latest change. I will help upload the new octant screenshot and provide the link.

mengdie-song commented:
I have uploaded to new screenshot to https://s3-us-west-2.amazonaws.com/downloads.antrea.io/static/tf_create_new.png. Please help replace tf_create with tf_create_new.png in this PR.

antoninbas commented:
@mengdie-song Thanks. I changed the TF create screenshot in the documentation (I renamed the file from `tf_create_new.png` to `tf_create.1.png` in S3, in case we need to upload new versions again in the future). You mentioned something in an earlier comment about the sample YAML definition needing to be updated. I took a quick look and it seems that it is still valid. Could you confirm?

jianjuns reviewed (commented):
LGTM

tnqn reviewed (approved):
LGTM

antoninbas commented:
/skip-all
